packages/error-debugging/dev-toolbar/src/toolbar/components/first-visit-hint.tsx

```
@@ -0,0 +1,127 @@
+            <button
+                class={cn(
+                    "w-full h-6 text-[0.6rem] font-bold uppercase tracking-[0.1em]",
+                    "border border-primary/30 bg-primary/6",
+                    "text-primary cursor-pointer",
+                    "hover:bg-primary/12 hover:border-primary/50",
+                    "transition-all duration-150 active:scale-[0.98]",
+                )}
+                onClick={dismiss}
+                type="button"
+            >
+                // got it
```

Biome (2.4.4)

[error] Wrap comments inside children within braces.

(lint/suspicious/noCommentText)